tests/test_TopicPartition.py

```diff
@@ -51,3 +51,10 @@ def __init__(self, topic_part_str):
     st = SubTopicPartition("topic2:920")
```

Comment:
Also add an integration test that verifies that the metadata can be commit():ed and then read back with committed(). Also check the null-case where metadata is not committed.